ci_cd/tasks/update_deps.py

```diff
@@ -268,9 +281,8 @@ def update_deps(  # pylint: disable=too-many-branches,too-many-locals,too-many-s
             hide=True,
         )
         package_latest_version_line = out.stdout.split(sep="\n", maxsplit=1)[0]
```

Comment:
The change from re.match to re.search and the pattern update to \S+ (any non-whitespace character) significantly improves the versatility in matching package versions. However, this pattern is now overly broad and may match unexpected strings. A more precise pattern might be warranted to reduce the risk of erroneous matches.

Reply:
The `\S+` cannot be changed, however, I can change it back to using `match` instead of `search`, which would be fine? Any precision comes later, when validating the two parts.